### ISSUE TITLE
keyboard focus on bookmark text causes slight content shift (border instead of outline?), and lacks padding

### ISSUE THREAD

**Issue by danielweck** [edrlab/thorium-reader#2153]
<img width="802" alt="Screenshot 2024-05-06 at 20 15 55" src="https://github.com/edrlab/thorium-reader/assets/625201/8dabe084-6926-4552-9772-9afe8945b560">

**Comment by danielweck** [comment]
Fixed via https://github.com/edrlab/thorium-reader/pull/2159
